HelpSource/Guides/GUI-Introduction.schelp

```diff
@@ -391,3 +391,6 @@ Routine{
 ::
 
 As mentioned above, using the GUI system is also not allowed in code performed directly in response to OSC messages (this includes functions given to all kinds of OSC responder classes). The same solutions as above apply:
+
+subsection:: GUI issues
+Some insights about common GUI issues can be found on the FAQ link::Guides/UserFAQ#Language (client) Issues#Language (client) Issues::
```

Comment:
The "Language (...) Issues" link describes the issue with timing, which explains the same thing as the paragraph right before this (*Caution: GUI and timing*). I think the `subsection` can be removed, the colon at the end of the previous sentence should be removed, and then the sentence could read:
> Another example for addressing this issue can be found in the FAQ \<link\>